packages/nns/src/types/governance_converters.ts

```diff
@@ -164,6 +164,12 @@ export interface ListProposalsRequest {
   // [ProposalStatus] for more information). If this list is empty, no
   // restriction is applied.
   includeStatus: Array<ProposalStatus>;
+
+  // Omits "large fields" from the response. Currently only omits the
+  // `logo` and `token_logo` field of CreateServiceNervousSystem proposals. This
+  // is useful to improve download times and to ensure that the response to the
```

Comment:
not related to review, curiosity question: is there any plan to provide the answer with some sort of streaming or pagination to solve the root cause?

Reply:
Not in their roadmap I believe.